core/eth/client_test.go

```diff
@@ -200,137 +198,3 @@ func TestCallerSubscriberClient_GetERC20Balance(t *testing.T) {
 		})
 	}
 }
-
```

Comment:
**Review notes:** these tests were moved verbatim to `FluxAggregator_test.go`.

Reply:
why no flux_aggregator_test.go?